pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -4902,6 +4902,23 @@ def test_lsts_from_time_with_only_unique():
     assert np.array_equal(full_lsts, uv.lst_array)
 
 
+@pytest.mark.filterwarnings("ignore:Altitude is not present in Miriad file")
+def test_lsts_from_time_with_only_unique_background():
+    """
+    Test `set_lsts_from_time_array` with only unique values is identical to full array.
+    """
+    miriad_file = os.path.join(DATA_PATH, "zen.2456865.60537.xy.uvcRREAA")
```

Comment:
More a note to myself: if this is merged in after #839, we'll want to use the corresponding uvh5 test file instead so we can still test this on Windows. (If this PR is merged first, I'll fix the other PR in my rebase).